examples/date-picker.amp.html

```diff
@@ -31,87 +31,62 @@
   <script async custom-element="amp-date-picker" src="https://cdn.ampproject.org/v0/amp-date-picker-0.1.js"></script>
   <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
   <script async src="https://cdn.ampproject.org/v0.js"></script>
 </head>
 <body>
-  <h1><code>amp-date-picker</code> example</h1>
-  <p>Experimental under the 'amp-date-picker' flag</p>
-  <h2><code>&lt;amp-date&gt;</code></h2>
-  <p>Try to pick a date. This is a single-date picker</p>
-  <button on="tap:sdp.setDate(date='2017-10-22')">Set date</button>
-  <button on="tap:sdp.clear">Clear</button>
-  <amp-date-picker
-    layout="container"
-    width="300"
-    height="300"
-    on="select:AMP.setState({date: event.date, dateType: event.id})"
-    locale="en"
-    format="YYYY-MM-DD"
-    number-of-months="1"
-    highlighted="2017-10-10 2017-10-12"
-    blocked="2017-10-10 2017-10-17 2017-10-19 2017-10-21 2018-01-29"
-    show-clear-date
-    keep-open-on-date-select
-    show-default-input-icon
-    id="sdp"
-  >
-    <input amp-date-placeholder placeholder="Pick a date!!" id="wow"/>
-    <template type="amp-mustache" date-template default>
-      {{^isBlocked}}{{^isHighlighted}}{{D}}{{/isHighlighted}}{{/isBlocked}}
-      {{#isBlocked}}:({{/isBlocked}}
-      {{#isHighlighted}}
-        {{D}}
-        <span class="highlight-corner"></span>
-      {{/isHighlighted}}
-    </template>
-    <template
-      type="amp-mustache"
-      date-template
-      dates="FREQ=WEEKLY;DTSTART=20170101T160000Z;INTERVAL=2;WKST=SU;BYDAY=TU 2017-10-30"
-      id="tacos1"
-    >
-      <span>🌮</span>
-      <span class="taco-tuesday"></span>
-      {{#isHighlighted}}
-        <span class="highlight-corner"></span>
-      {{/isHighlighted}}
-    </template>
-    <template type="amp-mustache" info-template>
-      <div
-        [text]="'You picked ' + date + '.' +
-        (dateType == 'tacos1' ? ' Happy Taco Tuesday!' : '')"
-      >You will see your chosen date here.</div>
-    </template>
-  </amp-date-picker>
-
+  <h1>amp-date-picker</h1>
   <h2><code>&lt;amp-date-range&gt;</code></h2>
   <p>Try to pick a range. This is a date-range picker</p>
-  <button on="tap:drp.setDates(startDate='2017-10-26',endDate='2017-11-04')">Set date range</button>
-  <button on="tap:drp.clear">Clear</button>
   <amp-date-picker
     type="range"
     id="drp"
+    reopen-picker-on-clear-date
     on="
     select:
       AMP.setState({
         dates: event.dates,
         tacoTuesdays: event.dates.filter(d => d.id == 'tacos2').length,
       })"
     locale="en"
-    format="Y-MM-DD"
-    blocked="2017-10-15 2017-09-17 2017-10-19 2017-10-21"
+    format="DD/MM/YYYY"
+    blocked="2018-02-26"
     highlighted="2018-01-29"
-    keep-open-on-date-select
     number-of-months="1"
+    day-size="50"
     show-clear-dates
+    keep-open-on-date-select
+    start-date-field="#start"
```

Comment:
What do you think about `x-date-input-selector`  or just `x-date-input`?

Reply:
I thought `field` was appropriate because not all sources of input are `<input>` tags, it could be a `contenteditable` or `textarea` etc. But if you feel like `*-field` is more confusing I'm glad to change it.

Reply:
I don't have a strong opinion. `field` usually confuses me with `fieldset` but looking at `amp-form` documentation, we actually use both `input` and `field` interchangeably :) regardless of `field` or `input` I suggest adding `selector` to the name for consistency as most other places we take a queryselector, attribute is called `selector`